app/models/manageiq/providers/vmware/cloud_manager/vm/operations/power.rb

```diff
@@ -0,0 +1,21 @@
+module ManageIQ::Providers::Vmware::CloudManager::Vm::Operations::Power
+  def validate_pause
+    validate_unsupported("Pause operation")
+  end
+
+  def raw_start
+    with_provider_object(&:power_on)
+  end
+
+  def raw_stop
+    with_provider_object(&:power_off)
```

Comment:
@gberginc after sending the command to the provider we should be updating the state in the db so that we don't have to wait for the next refresh to pick up the new power state.  See https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/openstack/cloud_manager/vm/operations/power.rb#L24 for some examples